src/server/HelixImportServer.js

```diff
@@ -63,6 +64,20 @@ export class HelixImportServer extends BaseServer {
     return url.toString();
   }
 
+  async _updateHeaders(headers, log) {
+    // eslint-disable-next-line no-underscore-dangle
+    const filePath = this._project._headersFile;
+    if (!filePath) return;
+    let cliHeaders;
+    try {
+      cliHeaders = await fs.readJSON(filePath);
+      // merge the headers defined with the CLI with those from the request
+      Object.assign(headers, cliHeaders);
+    } catch (error) {
+      log.error(`Failed to read or parse JSON file at ${filePath}: ${error.message}`);
```

Comment:
You probably don't need to catch the error here.

if reading the file fails, I as a developer would expect the cli to fail.

Reply:
good point, I removed the try-catch